doge_datagen/kafka_avro_sink_factory.py

```diff
@@ -9,19 +9,17 @@
 class KafkaAvroSinkFactory(object):
 
     def __init__(self,
```

Comment:
This change is not backwards compatible. Could you make conf an optional parameter and provide default value to existing values?
Then please add warning logging that this is deprecated.

Reply:
Corrected - conf param added as optional